codespell_lib/data/dictionary.txt

```diff
@@ -44588,6 +44588,7 @@ quotaions->quotations
 quoteed->quoted
 quotent->quotient
 quotin->quoting, quot in,
+quotted->quoted
 quottes->quotes
```

Comment:
Time permitting, could you add all verbal forms?
```suggestion
quottes->quotes
quotting->quoting
```

Reply:
Yep! I considered doing it, but since the variations were already slightly inconsistent, I didn't know if it would be welcome. I'll expand this PR with a few of these; starting with this suggestion.

Reply:
I've pushed a new revision with several more of these.